src/sentry/options/defaults.py

```diff
@@ -3467,3 +3467,12 @@
     default=True,
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
+
+# Controls whether we automatically disable certain detectors for
+# certain platforms on project creation.
+register(
```

Comment:
we don't have great guidelines on this but i would probably make this a flagpole flag rather than an option